packages/nimble-components/package.json

```diff
@@ -102,14 +105,15 @@
     "karma-source-map-support": "^1.4.0",
     "karma-sourcemap-loader": "^0.3.7",
     "karma-spec-reporter": "^0.0.36",
+    "karma-webkit-launcher": "^2.1.0",
     "karma-webpack": "^5.0.0",
+    "playwright": "^1.30.0",
     "prettier": "^2.0.0",
     "prettier-eslint": "^15.0.1",
     "prettier-eslint-cli": "^7.1.0",
     "puppeteer": "^19.6.0",
```

Comment:
The playwright npm install will already install chromium, firefox, and webkit right? Can we go ahead and purge puppeteer from the repo?

From a quick search it seems to only be used for it's chromium binary for angular and components so I expect that to be a straightforward change.

Reply:
Updated